assets/app/scripts/filters/resources.js

```diff
@@ -926,11 +926,24 @@ angular.module('openshiftConsole')
       if (!route.status.ingress) {
         return "Pending";
       }
-      var admittedCondition = routeIngressConditionFilter(route, 'Admitted');
-      if (!admittedCondition) {
+
+      var admittedConditions = [];
+      angular.forEach(route.status.ingress, function(ingress) {
+        var admitted = routeIngressConditionFilter(ingress, 'Admitted');
+        if (admitted) {
+          admittedConditions.push(admitted);
+        }
+      });
+
+      if (!admittedConditions.length) {
         return "Unknown";
       }
-      return admittedCondition.status === "True" ? "Admitted" : "Rejected";
+
+      if (_.some(admittedConditions, { status: "True" })) {
```

Comment:
I think the problem is we're trying to show multiple status as one value. We're only using this for preferred display routes. I suggest we just move the logic to route service.